cypress/integration/pages/articles/testsForCanonicalOnly.js

```diff
@@ -56,7 +56,7 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
           .scrollIntoView();
 
         cy.get('figure')
-          .eq(1)
+          .eq(2)
```

Comment:
Out of interest is this expecting to see two figure elements?

Reply:
Yeah, seeing we've added another figure element to wrap around the media player